examples/05-Retrieval-Model.ipynb

```diff
@@ -61,7 +61,8 @@
     "### Learning objectives\n",
```

Comment:
<p>Can we add a sentence WHY we want to do this? This is highly more efficient. Instead of calculating the candidate (item) tower for each example (query, aka user), we compute the output of the item tower once. This saves a lot of compute and is way more efficient</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>